tests/Import_scanner_test.py

```diff
@@ -53,33 +56,6 @@ def test_check_test_file(self):
                 logger.info(test)
         assert len(missing_tests) == 0
 
-    def test_check_for_doc(self):
-        driver = self.driver
-        driver.get("https://documentation.defectdojo.com/integrations/import/")
-        integration_index = integration_text.index("Integrations") + len("Integrations") + 1
-        usage_index = integration_text.index("Usage Examples") - len("Models") - 2
-        integration_text = integration_text[integration_index:usage_index].lower()
-        integration_text = integration_text.replace("_", " ").replace("-", " ").replace(".", "").split("\n")
-        acronyms = []
-        for words in integration_text:
-            acronyms += ["".join(word[0] for word in words.split())]
-
-        missing_docs = []
-        for tool in self.tools:
-            reg = re.compile(".*" + tool.replace("_", " ") + ".*")
-            if len(list(filter(reg.search, integration_text))) < 1:
-                if len(list(filter(reg.search, acronyms))) < 1:
-                    missing_docs += [tool]
-
-        if len(missing_docs) > 0:
-            logger.info("The following scanners are missing documentation")
-            logger.info("Names must match those listed in /dojo/tools")
-            logger.info("Documentation can be added here:")
-            logger.info("https://github.com/DefectDojo/django-DefectDojo/tree/dev/docs\n")
-            for tool in missing_docs:
-                logger.info(tool)
-        assert len(missing_docs) == 0
-
```

Comment:
Just commenting for other reviewers

This removal is good since there are dedicated tests to ensure parsers have docs that are upstream from this.